### ISSUE TITLE
Autocomplte query

### ISSUE THREAD

**Issue by gthayden** [elasticquent/Elasticquent#81]
I was wondering is there a way in whch i can use an autocomplete mapping, please excuse my dumbness but any takers on a fix?

**Comment by sahamilton** [comment]
Yes, this is something I really need ..ngram or partial word searching.  I'm new to Elasticsearch so not 100% clear on how this is done .. and especially not in Elasticquent.  any help would be appreciated.
